util/mkerr.pl

```diff
@@ -558,9 +558,9 @@ sub help
         $i =~ /^${lib}_F_(\S+)$/;
         $fn = $1;
         $fn = $ftrans{$fn} if exists $ftrans{$fn};
-        my $short = "    {ERR_PACK($pack_lib, $i, 0), \"$fn\"},\n";
-        if ( length($short) < 78 ) {
-            print OUT $short;
+        my $short = "    {ERR_PACK($pack_lib, $i, 0), \"$fn\"},";
+        if ( length($short) < 80 ) {
```

Comment:
Gonna be picky here...  `util/indent.pro` says `-l80`, that means `<= 80`, not `<80`.